cmd/tendermint/commands/lite.go

```diff
@@ -64,21 +81,32 @@ func init() {
 		"Maximum number of simultaneous connections (including WebSocket).")
 	LiteCmd.Flags().DurationVar(&trustingPeriod, "trusting-period", 168*time.Hour,
 		"Trusting period. Should be significantly less than the unbonding period")
-	LiteCmd.Flags().Int64Var(&trustedHeight, "trusted-height", 1, "Trusted header's height")
-	LiteCmd.Flags().BytesHexVar(&trustedHash, "trusted-hash", []byte{}, "Trusted header's hash")
+	LiteCmd.Flags().Int64Var(&trustedHeight, "height", 1, "Trusted header's height")
+	LiteCmd.Flags().BytesHexVar(&trustedHash, "hash", []byte{}, "Trusted header's hash")
+	LiteCmd.Flags().BoolVar(&verbose, "verbose", false, "Verbose output")
 }
 
 func runProxy(cmd *cobra.Command, args []string) error {
-	liteLogger := logger.With("module", "lite")
+	// Initialise logger.
+	logger := log.NewTMLogger(log.NewSyncWriter(os.Stdout))
+	var option log.Option
+	if verbose {
+		option, _ = log.AllowLevel("debug")
+	} else {
+		option, _ = log.AllowLevel("info")
+	}
+	logger = log.NewFilter(logger, option)
 
-	logger.Info("Connecting to the primary node...")
-	rpcClient, err := rpcclient.NewHTTP(chainID, primaryAddr)
+	chainID = args[0]
```

Comment:
Does Cobra handle argument validation, i.e. that there's exactly 1 argument?

Reply:
Yeah, that's what `Args: cobra.ExactArgs(1),` does.